modules/cnf-configuring-node-groups-for-the-numaresourcesoperator.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: PROCEDURE
+
+[id="cnf-configuring-node-groups-for-the-numaresourcesoperator_{context}"]
+= Configuring NUMA Resource Operator polling
+
+You can configure how the NUMA Resources Operator polls a `nodeGroup` for available NUMA resources by configuing the `spec.nodeGroups` specification in the `NUMAResourcesOperator` custom resource (CR). The configuration options are the following:
+
+* `infoRefreshMode`: Determines the trigger for polling updates to the API Server.
```

Comment:
I'm not a native english speaker, so bear with me: this seems to suggest somehow the updates could be done polling the apiserver (which is something a kubernetes component CAN do in theory!).  I'd rephrase in a different way in order to convey all the following
1. updates are done polling _the kubelet_
2. the trigger condition for the polling is controlled by this option
3. updates are thus _sent_ to the apiserver, regardless to the polling trigger